src/convertToObject.js

```diff
@@ -3,16 +3,27 @@
 /**
  * Implement convertToObject function:
  *
- * Function takes a string with styles (see an example in [stylesString.js](./stylesString.js))
+ * Function takes a string with styles
+ * (see an example in [stylesString.js](./stylesString.js))
  * and returns an object where CSS properties are keys
- * and values are the values of related CSS properties (see an exampl in [test file](./convertToObject.test.js))
+ * and values are the values of related CSS properties
+ * (see an exampl in [test file](./convertToObject.test.js))
  *
  * @param {string} sourceString
  *
  * @return {object}
  */
 function convertToObject(sourceString) {
-  // write your code here
+  const stylesCSS = {};
+
+  sourceString
+    .replace(/\r?\n/g, '')
+    .split(';')
+    .filter(item => item.length > 2)
+    .map(item => item.split(':'))
+    .forEach(item => (stylesCSS[item[0].trim()] = item[1].trim()));
```

Comment:
```suggestion
  sourceString
    .split(';')
    .filter(item => item.trim())
    .map(item => item.split(':'))
    .forEach(item => (stylesCSS[item[0].trim()] = item[1].trim()));
```

you can simplify your code by changing filter and getting rid of regular expression